src/main/java/DilemmaDetector/Simulator/RigidBody.java

```diff
@@ -7,6 +7,7 @@
     private Vector2 previousPosition = Vector2.zero();
     private Vector2 speed = Vector2.zero();
     private Vector2 acceleration= Vector2.zero();
+    private Double valueInDollars;
```

Comment:
Rigidbody nie powinno mieć wartości, jest to rzecz czysto do symulacji->detekcji kolizji, nie ma w odpowiedzialności posiadania kosztu

Reply:
Bardziej w aktorze np

Reply:
Racja mój błąd